bring/src/test/java/com/bobocode/petros/bring/factory/postprocessor/FieldInjectionBeanPostProcessorTest.java

```diff
@@ -0,0 +1,78 @@
+package com.bobocode.petros.bring.factory.postprocessor;
+
+import com.bobocode.petros.bring.context.ApplicationContext;
+import com.bobocode.petros.bring.context.domain.BeanReference;
+import com.bobocode.petros.bring.context.domain.BeanScope;
+import com.bobocode.petros.bring.exception.NoSuchBeanException;
+import com.bobocode.petros.bring.exception.NoUniqueBeanException;
+import com.bobocode.petros.bring.factory.postprocessor.mocks.Plane;
+import com.bobocode.petros.bring.factory.postprocessor.mocks.Car;
+import com.bobocode.petros.bring.factory.postprocessor.mocks.Engine;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+@ExtendWith(MockitoExtension.class)
+class FieldInjectionBeanPostProcessorTest {
+    @Mock
+    private ApplicationContext applicationContext;
+
+    private final Engine engine = new Engine();
+    private final Car car = new Car();
+    private final FieldInjectionBeanPostProcessor fieldInjectionBeanPostProcessor = new FieldInjectionBeanPostProcessor();
```

Comment:
MINOR (just FYI)I think you can use @InjectMocks annotation on fieldInjectionBeanPostProcessor, so it will be smth like:
```
@Mock
private ApplicationContext applicationContext;
@InjectMocks
private FieldInjectionBeanPostProcessor fieldInjectionBeanPostProcessor;
```
and you won't need setup() method at all